var/spack/repos/builtin/packages/diffsplice/package.py

```diff
@@ -17,6 +17,11 @@ class Diffsplice(MakefilePackage):
     version('0.1.2beta', 'a1df6e0b50968f2c229d5d7f97327336')
     version('0.1.1',     'be90e6c072402d5aae0b4e2cbb8c10ac')
 
+    def edit(self, spec, prefix):
+        if 'aarch64' in spec.architecture.target.lower():
```

Comment:
maybe even `if not 'amd64' in ...`?

Reply:
@ax3l should we merge as-is or wait for a change?

Reply:
The -m64 option is supported on x86, sparc and powerPC:
https://gcc.gnu.org/onlinedocs/gcc-9.1.0/gcc/Option-Summary.html#Option-Summary
So I write to delete '-m64' on aarch64 only.

Reply:
ok, thanks! Must have checked outdated docs then.